Extensions/PlayerAuthentication/playerauthenticationtools.ts

```diff
@@ -92,6 +107,38 @@ namespace gdjs {
       return _userId || null;
     };
 
+    /**
+     * Returns true if the game is registered, false otherwise.
+     * Useful to display a message to the user to register the game before logging in.
+     */
+    const isGameRegistered = (
```

Comment:
You could rename this `checkIfGameIsRegistered` to use `isGameRegistered` in other parts of the code.